OpenMEEG/libs/OpenMEEGMaths/include/matrix.h

```diff
@@ -222,7 +221,7 @@ namespace OpenMEEG {
         om_assert (istart+isize<=nlin() && jstart+jsize<=ncol());
 
         Matrix a(isize,jsize);
-        const int sz = static_cast<int>(isize);
+        const int sz = (int)isize;
```

Comment:
C casts are not type safe. C cast are mapped to 4 types in C++ static_cast, const_cast, dynamic_cast, and reinterpret_cast. dynamic_cast is of no interest here as we deal with POD. reinterpret_cast in C++ is the non type-safe cast (it is not relevant too here as it only deals with pointer and references). 

Quoting BS: C type cast should have been deprecated.... static_cast is the proper choice here. A bonus point if there is a check that the resulting number is not negative. To avoid very long lines, make a small inline function...

https://books.google.fr/books?id=EXfcAAAAQBAJ&pg=PA161&lpg=PA161&dq=C%2B%2B+C+type+casting+deprecated&source=bl&ots=2spW5eE_6W&sig=JCQDTqpEmAaEwkjP0yQvHISzPAM&hl=fr&sa=X&ved=0ahUKEwiegIXj9d3SAhWDHxoKHT6BB7gQ6AEITDAG#v=onepage&q=C%2B%2B%20C%20type%20casting%20deprecated&f=false